wagtailmodelchooser/views.py

```diff
@@ -62,7 +62,8 @@ def chooser(request, app_label, model_name, filter_name=None):
             raise Http404
         qs = filter_func(qs)
 
-    paginator, page = paginate(request, qs, per_page=10)
+    paginator = Paginator(qs, per_page=10)
+    page = paginator.get_page(request.GET.get('p'))
```

Comment:
can you change this to:

`page = paginator.page(request.GET.get('p', 1))`

Reply:
Done